docs-www/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@brainhubeu/gatsby-docs-kit": "3.0.8",
     "@brainhubeu/react-carousel": "^1.19.14",
-    "gatsby": "2.20.24",
+    "gatsby": "^2.24.2",
```

Comment:
we decided to pin the gatsby dependencies versions because there were some problems with upgrading

btw. I think I've already posted such a comment but it disappeared

Reply:
The older version didn't work for me. That's why I upgraded it

Reply:
so could you keep it pinned after upgrading?

Reply:
Why? If the new versions work with a new carousel why should we pin it?

Reply:
because it's likely that another version won't be working

Reply:
How do you know that? If the future version won't work for us we will pin the latest working one. Until then we should keep our dependencies up to date

Reply:
so renovate will upgrade it but it's good to test it before upgrading (merging a renovate PR)

Reply:
I'm ok with upgrading it by renovate and checking whether new versions didn't break anything

Reply:
so could you pin gatsby?

Reply:
OK, it's frozen in yarn.lock